testing/conftest.py

```diff
@@ -14,3 +15,34 @@ def create_signer(private_key):
     """
     private_key = Secp256k1PrivateKey.from_hex(private_key)
     return CryptoFactory(create_context('secp256k1')).new_signer(private_key)
+
+
+class Request:
+
+    def __init__(self):
+        self._params = None
+
+    @property
+    def params(self):
+        """
+        Substitute request params getter.
+        """
+        return self._params
+
+    @params.setter
+    def params(self, value):
+        """
+        Substitute request params setter.
+        """
+        self._params = {
+            'public_key_address': value,
```

Comment:
You will use this fixture (`Request`) one more time for another endpoint, so create a generic way to sent params instead of static `public_key_address`.

Reply:
Like `self._params = value`, but value is dictionary that is set from tests.

Reply:
@dmytrostriletskyi, done.